packages/core-data/src/selectors.ts

```diff
@@ -1145,10 +1145,22 @@ export function isPreviewEmbedFallback( state: State, url: string ): boolean {
 export function canUser(
 	state: State,
 	action: string,
-	resource: string,
+	resource: string | Record< string, any >,
```

Comment:
Should I swap this with `string | { kind: string; name: string; id?: EntityRecordKey }`?

Reply:
Yes. I'd also consider giving the type a name (like `EntityResource`) and the the type is `string | EntityResource`. Then it's more clear what this kind of parameter means.

Reply:
Updated in 3246e87a3c12cc31bb1e853a83f2d24d50cfd15e.